src/routes/Logout/components/LogoutPage/LogoutPage.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react';
+import { useFirebaseApp } from 'reactfire';
+import { useHistory } from 'react-router-dom';
+import {
+  makeStyles,
+  Container,
+  Typography,
+  Paper,
+  LinearProgress,
+} from '@material-ui/core';
+import { Helmet } from 'react-helmet';
+import styles from './LogoutPage.styles';
+
+const useStyles = makeStyles(styles);
+
+function LogoutPage() {
```

Comment:
What is the reason for the logout page instead of just having a logout button in the account dropdown?

Reply:
See my response to the [other comment](https://github.com/CV19Assist/app/pull/79#pullrequestreview-400617543).